Makefile

```diff
@@ -247,6 +247,7 @@ build-ubuntu-coverage:
 manifest:
 	@echo "===> Generating dev manifest for Antrea <==="
 	$(CURDIR)/hack/generate-manifest.sh --mode dev > build/yamls/antrea.yml
+	$(CURDIR)/hack/generate-manifest.sh --mode dev --full > build/yamls/antrea-full.yml
```

Comment:
Could we avoid checking this manifest in? Having more manifest files checked-in means that every PR that makes a small change to an Antrea resource leads to a big diff.

I feel like this manifest is only useful for testing and therefore we can generate in on the fly as part of the test script. We do the same thing for the "coverage" manifests in test-vmc.sh. For those we have a dedicated Makefile target (`make manifest-coverage`) but I think it's not necessary here: we can just call `generate-manifest.sh` directly from the test-vmc.sh script.

Reply:
Make sense. I have removed the generation in Makefile and moved it to `test-vmc.sh`.